spec/controllers/verify/finance_controller_spec.rb

```diff
@@ -142,7 +142,7 @@
 
         expect(@analytics).to have_received(:track_event).with(
           Analytics::IDV_FINANCE_CONFIRMATION, result
-        )
+        ).twice
```

Comment:
Good question. I think 2 events makes sense. If it doesn't add too much to this PR, I would say do it now, but I'll leave it up to you.